maquetado/login.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>login</title>
+  <link rel="stylesheet" type="text/css" href="application.css">
+</head>
+<body>
+  <div class="login-container">
+    <div class="login">
```

Comment:
Looks like you don't need this div. If you have a div just to wrap another div, you can combine them and do everything inside the first div

Reply:
The idea here is to use flex to center the login box vertically along the browser and also use padding-top since I can't use margin-top in the inner div.